test/e2e/containers_conf_test.go

```diff
@@ -258,6 +258,12 @@ var _ = Describe("Podman run", func() {
 		Expect(session.OutputToString()).To(Equal("0002"))
 	})
 
+	It("podman set network cmd options slirp options to allow host loopback", func() {
+		session := podmanTest.Podman([]string{"run", "--network", "slirp4netns", ALPINE, "ping", "-c1", "10.0.2.2"})
```

Comment:
@AkihiroSuda is the ip of `10.0.2.2` for the host, something defined by `slirp4netns` or by `podman`?

Reply:
This is slirp4netns's default value https://github.com/rootless-containers/slirp4netns/blob/master/slirp4netns.1.md#description